datasync/chaindatafetcher/chaindata_fetcher.go

```diff
@@ -287,22 +303,40 @@ func (f *ChainDataFetcher) Components() []interface{} {
 	return nil
 }
 
+func (f *ChainDataFetcher) setAPIs(apis []rpc.API) {
```

Comment:
This only sets `cn.PrivateDebugAPI`, which does not match its name.

Reply:
Is this an interface method?

Reply:
Nope, I will make another PR for this as well with the above comment.